src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1157,7 +1157,7 @@ private void InitializeNodeLibrary(IPreferences preferences)
                 // If the path has a .dll or .ds extension it is a locally imported library
                 if (extension == ".dll" || extension == ".ds")
                 {
-                    LibraryServices.ImportLibrary(path);
+                    LibraryServices.ImportLibrary(path, true);
```

Comment:
is this ever false?

Reply:
ImportLibrary is called in many places, this is the only place where the flag to not throw is explicitly set to true.